packages/constants/package.json

```diff
@@ -34,5 +33,6 @@
     "type": "git",
     "url": "git@github.com:leather-wallet/mono.git",
     "directory": "packages/constants"
-  }
+  },
+  "type": "module"
```

Comment:
Why at the bottom? This is a really important prop to see no?

Reply:
> Can we stick to kebab case naming for consistency 🙏🏼

Oh, yeap!

> Just to keep the import path cleaner, wouldn't @leather-wallet/ui/web be a nicer name?

+1 

> Thoughts on having web at root level and native on /native? In the same way it's react and react-native?

I would say it's better to keep it as `ui/web` and `ui/native` to be explicit

> Should the UI tool we use for web/native be in this package? Web doesn't have Text or View, so here the colocation doesn't really work. Does Panda work if we export Box via elsewhere?

Well i'd say that we would have instances when some components are very one platform specific anyways, so i'd say we should be probably fine for these basic components. I think having panda + restyle + storybook in one package is helpful as it keeps all building blocks close to each other. How can we approach it another way? 

> `theme-web/global` has extension specific code that we don't want on website for example, so that should remain in extension only.

Sure, later on we can separate it from theme-web and create a separate theme-extension.

> Why at the bottom? This is a really important prop to see no?

Syncpack orders all json keys alphabetically unless we explicitly set the order of that key with `sortFirst` in .syncpackrc. I'll add the type there now

Reply:
Thanks for comments. Sounds good 👍🏼 

> Sure, later on we can separate it from theme-web and create a separate theme-extension.

Only other comment is re this. If we have styles/logic that only impact the extension, imo they should live in extension not mono repo. For me the web/native terminology refers to environment, not product.